tests/integration/apps/test_command_line_apps.py

```diff
@@ -280,3 +288,67 @@ def test_upsert_quotes_with_valid_mapping_check_scaled_values(self):
         ]
 
         self.assertEqual(expected_value, test_values)
+
+    @parameterized.expand(
+        [
+            [
+                "outside_the_test_data_time",
+                flush_transactions.parse(
+                    args=[
+                        "testscope0001",
+                        "TestFlushPortfolio",
+                        "-s",
+                        "2020-02-20T00:00:00.0000000+00:00",
+                        "-e",
+                        "2020-02-28T23:59:59.0000000+00:00",
+                    ]
+                ),
+                6000,
+            ],
+            [
+                "inside_the_test_data_time",
+                flush_transactions.parse(
+                    args=[
+                        "testscope0001",
+                        "TestFlushPortfolio",
+                        "-s",
+                        "2020-02-10T00:00:00.0000000+00:00",
+                        "-e",
+                        "2020-02-28T23:59:59.0000000+00:00",
+                    ]
+                ),
+                0,
+            ],
+        ]
+    )
+    def test_flush_between_dates(self, _, args, expected_txn_count):
```

Comment:
- Is this a strong enough assertion to prove that you deleted the correct transactions?

- Is the date you are flushing for inclusive or exclusive?

- What happens if the Portfolio does not exist between the two provided dates?